### PR TITLE
Dmyers branch

### PR DESCRIPTION
**Pull request by Damon-Myers**

### CODE REVIEW

**Comment by Macbull** [comment]
please accept the CLA liscence !

**Comment by neilthemathguy** [comment]
Please accept the license.

**Comment by dmorina** [comment]
resolve the conflicts first please, thank you

**Comment by Damon-Myers** [comment]
Tests completed. Merge conflicts resolved. Please merge soon so that I don't have to keep doing this...

File: staticfiles/templates/contributors/home.html [hunk 1]
```diff
@@ -273,5 +291,10 @@ <h1 class="page-header"><small>Contributors</small></h1>
 </div>
 
 
+<<<<<<< HEAD
+ 		
+
+=======
+>>>>>>> 016bca849874ca8ff1a856ea98187babf3acef6e
```
Comment (dmorina) [inline staticfiles/templates/contributors/home.html]: fix these please

File: staticfiles/js/crowdsource.routes.js [hunk 2]
```diff
@@ -164,10 +165,20 @@
     .when('/contributors/vineet', {
       templateUrl: '/static/templates/contributors/vineet.html'
     })
+<<<<<<< HEAD
```
Comment (dmorina) [inline staticfiles/js/crowdsource.routes.js]: nope

**Pull request merged**